staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -760,6 +760,8 @@ type VirtualMachineInstanceMigrationState struct {
 
 	// Indicates the migration completed
 	Completed bool `json:"completed,omitempty"`
+	// Indicates that the migration has been finalized
+	Finalized bool `json:"finalized,omitempty"`
```

Comment:
What is the difference with completed? It is a bit confusing

Reply:
The finalization is performed by the target virt-handler when the migration has completed, this involves things like hotplugging cpus and memory.

Completed is when the migration itself has completed.

Reply:
@acardace To be more specific, why adding this fields solves the issue?

Reply:
The field is used to make sure we do finalization once and only once when the migration has completed, I didn't see any easier way to do this as the completion event is only processed by the source virt-handler.

Of course suggestions are welcome.